packages/lodestar/src/cli/commands/validator.ts

```diff
@@ -19,31 +19,34 @@ export class ValidatorCommand implements CliCommand {
 
   public register(commander: CommanderStatic): void {
     const logger: ILogger = new WinstonLogger();
-
     const command = commander
       .command("validator")
       .description("Start lodestar validator")
-      .option(`-l, --loggingLevel [${Object.values(LogLevel).join("|")}]`, "Logging level")
+      .option("-l, --loggingLevel [chain=debug, network=trace, database=warn]",
```

Comment:
there is no modules here, just let users pick logging level for validator module and pass it on as option